packages/purpose-process/test/getPurposeById.test.ts

```diff
@@ -239,6 +252,215 @@ describe("getPurposeById", () => {
       });
     }
   );
+
+  it("should get the purpose with the risk analysis form if the requester is an e-service delegated consumer", async () => {
```

Comment:
Could you also add some tests to check that the purpose Consumer/Producer/Producer Delegate can retrieve a purpose created by a consumer delegate? 
For the other calls this was not possible, only the delegate was able to act on purposes created by them. But for this call and the getPurposes it's a valid case, the Consumer can get delegated purposes.